### PR TITLE
OSS Bucket module

### PR DESCRIPTION
**Pull request by coderall**
new Terraform OSS Bucket module implementation

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
+.DS_Store   
+.idea   
 
-# others
-*.bak
+# others    
+*.bak   
 *.bk
-
-
```
Comment (xiaozhu36) [inline .gitignore]: I think the file .gitignore should not be modified.

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
-# terraform-alicloud-oss-bucket
\ No newline at end of file
+Alicloud OSS Terraform Module On Alibaba Cloud 
+terraform-alicloud-oss
```
Comment (xiaozhu36) [inline README.md]: terraform-alicloud-oss-bucket?

File: README.md [hunk 3]
```diff
@@ -1 +1,62 @@
...
+
+Usage
+-----
+You can use this in your terraform template with the following steps.
+
+1. Adding a module resource to your template, e.g. main.tf
+
+    ```
+    module "oss-bucket" {
+        source     = "../../"
```
Comment (xiaozhu36) [inline README.md]: Please refer to other module and set the correct source address.

File: examples/create_oss_bucket/main.tf [hunk 4]
```diff
@@ -0,0 +1,47 @@
+provider "alicloud" {
+  region = "cn-beijing"
+}
+
```
Comment (xiaozhu36) [inline examples/create_oss_bucket/main.tf]: Please remove the provider and use the default config.

File: variables.tf [hunk 5]
```diff
@@ -0,0 +1,29 @@
+variable "bucket_name" {
+  type    = "string"
+  default = ""
```
Comment (xiaozhu36) [inline variables.tf]: Please remove the bucket_name default.

File: .gitignore (hunk 1 above)
Comment (coderall) [reply .gitignore]: ok

File: README.md (hunk 2 above)
Comment (coderall) [reply README.md]: fix

File: variables.tf (hunk 5 above)
Comment (coderall) [reply variables.tf]: fix

File: examples/create_oss_bucket/main.tf (hunk 4 above)
Comment (coderall) [reply examples/create_oss_bucket/main.tf]: fix

**Pull request merged**